app/models/account.rb

```diff
@@ -46,10 +46,18 @@ def self.canonical_cname(cname)
     cname
   end
 
+  # Make all the account specific connections active
   def switch!
     solr_endpoint.switch! if solr_endpoint
     fcrepo_endpoint.switch! if fcrepo_endpoint
     redis_endpoint.switch! if redis_endpoint
+    load_workflows
+  end
+
+  # Ensure that some workflows have been loaded into the database
+  def load_workflows
+    return if Sipity::Workflow.any?
```

Comment:
@jcoyne Why do we need this check? If I have, say, 2 workflows available and I run the importer *n* times, I will still wind up with 2 `Sipity::Workflow`s. Performance related, I guess?